keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -0,0 +1,733 @@
+# Object Storage Support
+
+## Table of Contents
+
+<!-- toc -->
+- [Release Signoff Checklist](#release-signoff-checklist)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [User Stories](#user-stories)
+    - [Admin](#admin)
+    - [User](#user)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Vocabulary](#vocabulary)
+- [Proposal](#proposal)
+  - [APIs](#apis)
+    - [Storage APIs](#storage-apis)
+      - [BucketRequest](#bucketrequest)
+      - [Bucket](#bucket)
+      - [BucketClass](#bucketclass)
+    - [Access APIs](#access-apis)
+      - [BucketAccessRequest](#bucketaccessrequest)
+      - [BucketAccess](#bucketaccess)
+      - [BucketAccessClass](#bucketaccessclass)
+    - [App Pod](#app-pod)
+    - [Topology](#topology)
+- [Object Relationships](#object-relationships)
+- [Workflows](#workflows)
+  - [Create Bucket](#create-bucket)
+  - [Sharing COSI Created Buckets](#sharing-cosi-created-buckets)
+  - [Delete Bucket](#delete-bucket)
+  - [Grant Bucket Access](#grant-bucket-access)
+  - [Revoke Bucket Access](#revoke-bucket-access)
+  - [Delete BucketAccess](#delete-bucketaccess)
+  - [Setting Access Permissions](#setting-access-permissions)
+    - [Dynamic Provisioning](#dynamic-provisioning)
+    - [Static Provisioning](#static-provisioning)
+- [gRPC Definitions](#grpc-definitions)
+  - [ProvisionerGetInfo](#provisionergetinfo)
+  - [ProvisonerCreateBucket](#provisonercreatebucket)
+  - [ProvisonerDeleteBucket](#provisonerdeletebucket)
+  - [ProvisionerGrantBucketAccess](#provisionergrantbucketaccess)
+  - [ProvisionerRevokeBucketAccess](#provisionerrevokebucketaccess)
+- [Test Plan](#test-plan)
+- [Graduation Criteria](#graduation-criteria)
+  - [Alpha](#alpha)
+- [Alternatives Considered](#alternatives-considered)
+  - [Add Bucket Instance Name to BucketAccessClass (brownfield)](#add-bucket-instance-name-to-bucketaccessclass-brownfield)
+    - [Motivation](#motivation-1)
+    - [Problems](#problems)
+<!-- /toc -->
+
+# Release Signoff Checklist
+
+- [X] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [X] (R) Graduation criteria is in place
```

Comment:
This one probably should not be checked yet as I see you only have "Alpha" there.

Reply:
Fixed